tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers.ts

```diff
@@ -171,20 +173,33 @@ const reducer = createReducer(
     actions.debuggerRunsLoaded,
     (state: DebuggerState, {runs}): DebuggerState => {
       const runIds = Object.keys(runs);
+      const activeRunChanged = runIds.length > 0 && state.activeRunId === null;
       return {
         ...state,
+        lastNewPollDataTime: activeRunChanged
+          ? Date.now()
+          : state.lastNewPollDataTime,
         runs,
         runsLoaded: {
           state: DataLoadState.LOADED,
           lastLoadedTimeInMs: Date.now(),
         },
-        activeRunId: runIds.length ? runIds[0] : null,
+        activeRunId: runIds.length > 0 ? runIds[0] : null,
         // TODO(cais): Handle multiple runs. We currently assumes there is only
         // one run, which is okay because the backend supports only one run
         // per experiment.
       };
     }
   ),
+  on(
+    actions.debuggerDataPoll,
```

Comment:
I totally understood this action wrongly when I only read the contract described in the action/type/variable-name (the documentation is really lacking so I went based off of the variable naming). This should read "debuggerDataPollTriggered" and the reducer state should read `lastDataPollTriggerTimeMs`.

Reply:
Done renaming this field and the related one. I gave it a slightly different (and shorter) name: `lastDataPollOnsetTimeMs`